src/logger.ts

```diff
@@ -0,0 +1,28 @@
+import * as node from '@api3/airnode-node';
+
+type LogOptionsOverride = 'meta' | 'additional';
+
+export const buildLogOptions = (
+  override: LogOptionsOverride,
+  fields: { [key: string]: unknown },
+  baseLogOptions: node.LogOptions
+): node.LogOptions => {
```

Comment:
Right but I think it provides better typing on the caller. When I hover over the variable being assigned the result of calling this function it shows a dynamic type with all the props instead of node.LogOptions if I remove it. If you still feel like it is redundant the I'll remove it anyways.